testInput/json/key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
A candidate for env as well.

Reply:
Yes we can remove now. we move then to base64 encoded